Tests/FoundationInternationalizationTests/PredicateInternationalizationTests.swift

```diff
@@ -14,17 +14,15 @@
 import TestSupport
 #endif
 
-// Predicate does not back-deploy to older Darwin versions
-#if FOUNDATION_FRAMEWORK || os(Linux) || os(Windows)
+#if FOUNDATION_FRAMEWORK
```

Comment:
It doesn't matter _that_ much since the only test in this file is `FOUNDATION_FRAMEWORK`-only, but there may definitely be tests in this test case that are applicable to linux/windows too. Thoughts on just dropping this declaration and leaving the `#if FOUNDATION_FRAMEWORK` in place below to make that clearer?

Reply:
Ahh sounds good! Let's do that.